exercises/go-counting/canonical-data.json

```diff
@@ -147,26 +147,17 @@
       }
     },
     {
-      "description": "Open corner territory on 9x9 board",
-      "property": "territory",
+      "description": "Two region rectangular board",
+      "property": "territories",
       "input": {
         "board": [
-          "  B   B  ",
-          "B   B   B",
-          "WBBBWBBBW",
-          "W W W W W",
-          "         ",
-          " W W W W ",
-          "B B   B B",
-          " W BBB W ",
-          "   B B   "
-        ],
-        "x": 0,
-        "y": 8
+          " B "
+        ]
       },
       "expected": {
-        "owner": "NONE",
-        "territory": [[2, 7], [2, 8], [1, 8], [0, 8], [0, 7]]
+        "territoryBlack": [[0, 0], [0, 2]],
```

Comment:
inconsistent: This uses `[y, x]` coordinate instead of `[x, y]`, since a y coordinate of 2 does not exist (the x coordinate should be 2).